toolkits/global/packages/global-card/demo/main.scss

```diff
@@ -1,5 +1,5 @@
-@import '../node_modules/@springernature/brand-context/default/scss/core.scss';
-@import '../node_modules/@springernature/brand-context/default/scss/enhanced.scss';
+@import '../node_modules/@springernature/brand-context/springer/scss/core.scss';
```

Comment:
Out of curiosity, will the 'demo' for all components be the Springer brand, or can it vary?